esterel-rhombus-lib/esterel/full.rhm

```diff
@@ -16,6 +16,11 @@ export:
   loop
   await
   every
+  sustain
+
+fun
+| sustain(s): full.sustain(s)
+| sustain(s, v): full.sustain(s, v) // this still needs to be tested
   
```

Comment:
Probably good to include the change and the tests together in a single pull request